modules/local/plot_homer_annotatepeaks.nf

```diff
@@ -27,7 +27,7 @@ process PLOT_HOMER_ANNOTATEPEAKS {
         -p $prefix \\
         $args
 
-    find ./ -type f -name "*.txt" -exec cat {} \\; | cat $mqc_header - > ${prefix}.summary_mqc.tsv
+    find ./ -type f -name "*summary.txt" -exec cat {} \\; | cat $mqc_header - > ${prefix}.summary_mqc.tsv
```

Comment:
Fix bug noticed whilst testing on Azure where MultiQC sections were being rendered in the wrong order in the report. Still need to test but I suspect it is happening due to this line. Basically the script is pickup up all files ending in `.txt` because they are copied into the work directory and hence appending the contents to `summary_mqc.tsv` which then doesn't render in MultiQC properly.

Reply:
Great catch. I can see the `*peaks.annotatePeaks.txt` files in the working directory which would have been picked up in error!